src/app/app.component.scss

```diff
@@ -0,0 +1,14 @@
+$header-height: 56px;
+$footer-height: 80px;
+$menu-width: 240px;
```

Comment:
Hi, could you please use here 'vh' and 'vw' instead of 'px'? We have to make sure that it looks beatiful like on my 1920 * 1080

Reply:
bad idea bro